elements/rh-sass/mixins/_mixins.scss

```diff
@@ -68,6 +68,10 @@
   }
 }
 
+@mixin rh-arrow-color($color) {
```

Comment:
Why build a mixin for this one property?  Is the goal to evolve it to take care of other styling as well?